src/components/stopPoster/stopPoster.css

```diff
@@ -21,7 +21,7 @@
 }
 
 .columns {
-    flex: 1 0 auto;
+    flex: 1 1 auto;

```

Comment:
Allow the columns container to shrink, otherwise it overflow beyond the footer and makes it impossible to measure the column height.